auction-app-backend/src/main/java/com/example/auctionapp/request/ProductImageAddRequest.java

```diff
@@ -0,0 +1,39 @@
+package com.example.auctionapp.request;
+
+import com.example.auctionapp.entity.ProductImageEntity;
+
+import java.util.UUID;
+
+public class ProductImageAddRequest {
```

Comment:
We have ProductImageAddRequest, but we aren't using it
Avoid adding code that you don't need.